rmgpy/rmg/main.py

```diff
@@ -898,6 +902,27 @@ def execute(self, **kwargs):
                 
                 plot_sensitivity(self.outputDirectory, index, reactionSystem.sensitiveSpecies)
 
+        # Check all core reactions (in both directions) for collision limit violation
+        logging.info("Checking core reactions for collision rate limit violators...")
+        violators = []
+        for rxn in self.reactionModel.core.reactions:
+            f_violator, r_violator = rxn.check_collision_limit_violation(t_min=self.Tmin, t_max=self.Tmax,
+                                                            p_min=self.Pmin, p_max=self.Pmax)
+            if f_violator is not None:
+                violators.append(f_violator)
+            if r_violator is not None:
+                violators.append(r_violator)
+        if len(violators):
+            logging.info("\n")
+            logging.warning("{0} CORE reactions violate the collision rate limit!"
+                            " See the 'collision_rate_violators.log' for details.\n\n".format(len(violators)))
+            with open('collision_rate_violators.log', 'w') as violators_f:
```

Comment:
Do we need to check if a file with this name already exists? Otherwise, this file will get overwritten if the user restarts the job (that might be okay for this file, I am just checking because I know that we do this for the RMG log file)

Reply:
I think that since the model itself (chemkin/cantera formats) is being overwritten, and this output directly relates to the generated model, it's ok the overwrite the collision violators log too. A related question, though, is what if the user re-runs in the same folder, and only the first run generates the collision violators log file... I guess I should delete it if it exists and no violators were found?